src/wnt.cpp.props

```diff
@@ -0,0 +1,7 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemDefinitionGroup>
+    <ClCompile>
+      <TreatWarningAsError>true</TreatWarningAsError>
```

Comment:
We should have put this into `windows.undocked.props` instead of making a new one.

Reply:
I wasn't sure if undocked was intentionally not treating warnings as errors - is that the recommendation? If so, I'll migrate it.

Reply:
We definitely want to standardize on as secure as possible with undocking.